crates/matrix-sdk-sqlite/src/event_cache_store.rs

```diff
@@ -532,9 +539,11 @@ impl EventCacheStore for SqliteEventCacheStore {
                     }
                 }
 
+                txn.commit()?;
+
                 Ok(())
             })
-        .await?;
+        .await.unwrap()?;
```

Comment:
What's that unwrap about?

Reply:
Copied from the implementation of `with_transaction()`; the only valid error this can return is because an underlying spawned task has panick'd, which is a valid reason to panic ourselves.